src/test/java/org/broadinstitute/hellbender/cmdline/GATKPlugin/GATKAnnotationPluginDescriptorUnitTest.java

```diff
@@ -651,4 +658,37 @@ public void testOverridingInstancesWithGetInstance() throws InstantiationExcepti
         }
     }
 
+    @CommandLineProgramProperties(summary="test tool to check annotation loading",
+            oneLineSummary = "test tool to check annotation loading",
+            programGroup = TestProgramGroup.class,
+            omitFromCommandLine = true)
+    public static class TestAnnotationsTool extends GATKTool {
+        @Argument(fullName = StandardArgumentDefinitions.OUTPUT_LONG_NAME)
+        String output;
+
+        @Override
+        public boolean useVariantAnnotations(){
+            return true;
+        }
+
+        @Override
+        public void traverse() {
+            String annotations = makeVariantAnnotations().stream()
+                    .map(a -> a.getClass().getSimpleName())
+                    .collect(Collectors.joining("\n"));
+            IOUtil.writeText(annotations, new File(output));
+        }
+    }
+
+    @Test
+    public void testConfigFileControlsAnnotationPackages() throws IOException {
+        final File output = createTempFile("annotations", "txt");
+        final File configFile = new File("src/test/resources/org/broadinstitute/hellbender/cmdline/GATKPlugin/changePluginPackages.properties");
```

Comment:
should probably use the default test dir constants.

Reply:
yes